collections/misc/collstats.php

```diff
@@ -764,57 +765,59 @@ function changeCollForm(f){
 										<legend><?php echo $LANG['GENERAL_STATISTICS']?></legend>
 										<ul class="stats-display-ul">
 											<?php
-											echo "<li>";
-											echo ($results['SpecimenCount'] ? number_format($results['SpecimenCount']) : 0) . " " . $LANG['OCC_RECORDS'];
-											echo "</li>";
-											echo "<li>";
-											$percGeo = '';
-											if($results['SpecimenCount'] && $results['GeorefCount'] && $results['SpecimenCount'] !== 0){
-												try {
-													$percGeo = (100* ($results['GeorefCount'] / $results['SpecimenCount']));
-												} catch (Exception $e) {
-													error_log('Exception: ' . $e->getMessage());
-												}
-											}
-											echo ($results['GeorefCount'] ? number_format($results['GeorefCount']) : 0) . ($percGeo ? " (" . ($percGeo>1 ? round($percGeo) : round($percGeo,2)) . "%)" : '') . " " . $LANG['GEOREFERENCED'];
-											echo "</li>";
-											echo "<li>";
-											$percImg = '';
-											if($results['SpecimenCount'] && $results['TotalImageCount'] && $results['SpecimenCount'] !== 0){
-												try {
-													$percImg = (100* ($results['TotalImageCount'] / $results['SpecimenCount']));
-												} catch (Exception $e) {
-													error_log('Exception: ' . $e->getMessage());
-												}
-											}
-											echo ($results['TotalImageCount'] ? number_format($results['TotalImageCount']) : 0) . ($percImg ? " (" . ($percImg>1 ? round($percImg) : round($percImg,2)) . "%)" : '') . " " . $LANG['OCCS_IMAGED'];
-											echo "</li>";
-											echo "<li>";
-											$percId = '';
-											if($results['SpecimenCount'] && $results['SpecimensCountID'] && $results['SpecimenCount'] !== 0){
-												try {
-													$percId = (100* ($results['SpecimensCountID'] / $results['SpecimenCount']));
-												} catch (Exception $e) {
-													error_log('Exception: ' . $e->getMessage());
+												if ($results){
```

Comment:
maybe check for 'SpecimentCount' and 'GeorefCount' array key in $results?

Reply:
I dunno... it depends on whether it's possible to ever have at $results array without these keys, I suppose. I leave it up to you!